### PR TITLE
#128 - add SpaceAfterExclamationMark transformation

### PR DESCRIPTION
**Pull request by Lykegenes**
fixes issue #128

### CODE REVIEW

File: src/tests/372-indent-phpdoc.out-got [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?php
```
Comment (ccirello) [inline src/tests/372-indent-phpdoc.out-got]: Why did you commit this file?

Comment (Lykegenes) [inline src/tests/372-indent-phpdoc.out-got]: By mistake. Shouldn't this be in a `.gitignore` file?

Comment (ccirello) [inline src/tests/372-indent-phpdoc.out-got]: If you run correctly, this file doesn't even show up in the end.

**Comment by ccirello** [comment]
Please squash into a single commit, after that I shall merge this PR.

**Comment by Lykegenes** [comment]
There was a GitHub update earlier today allowing Squash & Merge. See the [blog post here](https://github.com/blog/2141-squash-your-commits). But it's April fools, so I hope it works!
The option shows up after you click the `Merge Pull Request` button.

**Pull request merged**